src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -32,6 +35,24 @@ async def _upload_bytes(
         expiration_policy: UndefinedOr[ExpirationPolicyAlias] = UNDEFINED,
         timeout: float = 60,
     ) -> FileTypeT:
+        """Uploads a byte array as a file.
+
+        :param data: The byte data to upload.
+        :param name: The name of the file.
+            Defaults to UNDEFINED.
+        :param description: A description of the file.
+            Defaults to UNDEFINED.
+        :param mime_type: The MIME type of the file.
+            Defaults to UNDERFINED.
+        :param labels: Labels associated with the file.
+            Defaults to UNDEFINED.
+        :param ttl_days: Time-to-live in days for the file.
+            Defaults to UNDEFINED.
+        :param expiration_policy: Expiration policy for the file.
```

Comment:
I want to describe here something about scope of definition, like you could pass "static" or "since_last_active" string, and maybe here some reference to some basic article about expiration would be appropriate (which is not created yet)